docker/dockerfile-build-ubuntu-rock

```diff
@@ -0,0 +1,48 @@
+# This Dockerfile provides a starting point for a ROCm installation of hipCaffe.
+
+# Parameters related to building hip
+ARG base_image
+
+FROM ${base_image}
+LABEL maintainer="kent.knox@amd"
+
+USER root
+ARG user_uid
+
+# Install dependent packages
+# Dependencies:
+# * hcc-config.cmake: pkg-config
+# * tensile: python2.7, python-pip

```

Comment:
new dependencies are python3 and rock-dkms

Reply:
I added rock-dkms to this dockerfile. python3 is already added, but python 2.7 should be removed as a dependency, for sure. Not sure this is worth another run in CI though...

Reply:
I should also note that dockerfile-build-ubuntu-rock is word-for-word the same file as dockerfile-build-rocm-terminal. To use the shared library, the build image must be named "dockerfile-build-ubuntu-rock"

Reply:
I mean add any new dependencies you have in the comments, not just silently add it in the code. People reading through will get confused if you just add things without documentation

Reply:
We should probably just delete this comment since it just duplicates the list of packages below.

Reply:
Yeah, I've marked it as resolved for that reason. You've taken care of this file anyway in your PR